src/es5processor.ts

```diff
@@ -323,11 +322,17 @@ class ES5Processor extends Rewriter {
     switch (call.expression.kind) {
       case ts.SyntaxKind.Identifier:
         const ident = call.expression as ts.Identifier;
-        if (ident.text !== '__export') return null;
+        // TS 2.4 compatibility: accept three leading underscores
```

Comment:
Nit: can we make this comment something that's more obviously a greppable tag? `// TS_24_COMPAT: ...`?

Reply:
sure, but do you think we can remember to follow that convention when we add backwards compatibility logic next time?

Reply:
If we do use a more obvious tag like name, we at least have a fighting chance :-)